target/scripts/check-for-changes.sh

```diff
@@ -59,6 +59,8 @@ function _check_for_changes
     # While some config changes may be properly applied by Postfix or Dovecot
     # via their 'reload' commands; some may require restarting?:
     _log_with_date 'debug' 'Restarting services due to detected changes'
+
+    _reload_amavis
```

Comment:
Only do so if Amavis service is active:

```suggestion
    [[ ${ENABLE_AMAVIS} -eq 1 ]] && _reload_amavis
```